src/helper/stream/index.ts

```diff
@@ -0,0 +1,48 @@
+import type { Context } from '../../context'
+import { StreamingApi } from '../../utils/stream'
+
+interface SSEMessage {
+  data: string
+  event?: string
+  id?: string
+}
+
+class SSEStreamingApi extends StreamingApi {
+  constructor(originalStream: StreamingApi) {
+    super(originalStream['writable']) // Accessing the private writable property - normally not recommended but works for this specific solution.
+  }
+
+  async writeSse(message: SSEMessage) {
```

Comment:
You can write `writeSse` and `writeSSE`, what do you think? I think `writeSSE` is better to unify with others.

Reply:
fixed